### PR TITLE
Add settings for Travis CI

### PR DESCRIPTION
**Pull request by giginet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +41,19 @@ For information on how to use CocoaPods, please refer to the official CocoaPods
 
 Before you write your Pact definitions, you have to configure the mock server.
 
-Edit your test scheme and add the following to the `Pre-actions` section. (You might have to make changes to the PATH setting.)
+First, add `Gemfile` on project root
```
Comment (vincentisambart) [inline README.md]: s/on/to

File: README.md [hunk 2]
```diff
@@ -38,11 +41,19 @@ For information on how to use CocoaPods, please refer to the official CocoaPods
...
+
+```ruby
+source "https://rubygems.org"
+
+gem "pact-mock_service"
+```
+
+Next, edit your test scheme and add the following to the `Pre-actions` section. (You might have to make changes to the PATH setting.)
```
Comment (vincentisambart) [inline README.md]: s/Next/Then

File: Tests/Phakchi/IntegrateTest.swift [hunk 3]
```diff
@@ -338,9 +338,15 @@ class IntegrateTestCase: XCTestCase {
...
     override func tearDown() {
         super.tearDown()
-        let exp = expectationWithDescription("session is closed")
-        session.close {
-            exp.fulfill()
+
+        let expectation = expectationWithDescription("session is closed")
+
+        // workaround to pass test on CI
+        let delayTime = dispatch_time(DISPATCH_TIME_NOW, Int64(1 * Double(NSEC_PER_SEC)))
```
Comment (vincentisambart) [inline Tests/Phakchi/IntegrateTest.swift:345]: ないとどういう問題が起きるんだろう

Comment (giginet) [reply Tests/Phakchi/IntegrateTest.swift:345]: 実行タイミングによって転けたり通ったりします。
delay入れると転けなくなるようだ･･････

https://travis-ci.org/cookpad/Phakchi/jobs/159241184#L413

**Comment by giginet** [comment]
#2 contains this changes, so I close this.

File: README.md [hunk 4]
```diff
@@ -109,6 +120,17 @@ class RecipeClientPact: XCTestCase {
         })
         waitForExpectationsWithTimeout(5, handler: nil)
     }
+
+    override func tearDown() {
+        super.tearDown()
```
Comment (vincentisambart) [inline README.md:125]: `super.tearDown()` should probably at the end of `tearDown()`.

File: Tests/Phakchi/IntegrateTest.swift (hunk 3 above)
Comment (vincentisambart) [reply Tests/Phakchi/IntegrateTest.swift:345]: Shouldn't you just increase the timeout duration given to `waitForExpectationsWithTimeout`?

**Comment by giginet** [comment]
I removed such a bad workaround.
I modified timing to launch mock servers

https://github.com/cookpad/Phakchi/pull/2/commits/2c81920f7480a7b6fb4597e9fbacf8291f82bad1

**Pull request closed without merging**